gremlin-python/src/main/python/tests/driver/test_client.py

```diff
@@ -47,11 +48,12 @@ def test_connection(connection):
 
 def test_client_message_too_big(client):
     try:
-        client = Client("http://localhost", 'g', max_content_length=1024)
-        client.submit("1+1").all().result()
+        client = Client('ws://localhost:45940/gremlin', 'g', max_content_length=1024)
```

Comment:
I see that this url `ws://localhost:45940/gremlin` is repeatedly used throughout the test suite, perhaps we should abstract it to a constant so it's easier to change in the future?

Reply:
done in https://github.com/lyndonbauto/tinkerpop/pull/118/commits/7d7464e81c5c6c3888fc6e9ea1c1a6633f6f0eed